pkg/rbac/userData.go

```diff
@@ -153,9 +153,32 @@ func (user *UserDataCache) userHasAllAccess(ctx context.Context, cache *Cache) (
 
 		cache.shared.mcCache.lock.Lock()
 		defer cache.shared.mcCache.lock.Unlock()
-		user.ManagedClusters = cache.shared.managedClusters
+		user.ManagedClusters = map[string]struct{}{"*": {}}
 		user.clustersCache.updatedAt = time.Now()
 		user.csrCache.err, user.nsrCache.err, user.clustersCache.err = nil, nil, nil
+		klog.V(5).Infof("User %s with uid %s has *list* access to all managed cluster resources.",
+			user.userInfo.Username, user.userInfo.UID)
+
+		return true, nil
+	} else if user.userAuthorizedListSSAR(ctx, impersClientSet,
+		"get", "searches", "searches/allManagedData") {
+		user.csrCache.lock.Lock()
+		defer user.csrCache.lock.Unlock()
+		user.CsResources = []Resource{}
+		user.csrCache.updatedAt = time.Now()
+
+		user.nsrCache.lock.Lock()
+		defer user.nsrCache.lock.Unlock()
+		user.NsResources = map[string][]Resource{}
+		user.nsrCache.updatedAt = time.Now()
+
+		cache.shared.mcCache.lock.Lock()
+		defer cache.shared.mcCache.lock.Unlock()
+		user.ManagedClusters = map[string]struct{}{"*": {}}
+		user.clustersCache.updatedAt = time.Now()
+		user.csrCache.err, user.nsrCache.err, user.clustersCache.err = nil, nil, nil
+		klog.V(5).Infof("User %s with uid %s has *get* access to all managed cluster resources.",
```

Comment:
What's the difference between list and get? 
Maybe we should log "user is authorized to search/allManagedData which gives access to a all managed cluster resources."

Reply:
Used 'get' as we gave that permission in the cluster role here. https://github.com/stolostron/search-v2-operator/blob/7f9bc36bd8a5dc35863c35b73dc43b91737d74db/controllers/create_rolesbindings.go#L214
'get' gets a resource by name while list lists all based on the selector, is what I think.  @jlpadilla